app/models/ability.rb

```diff
@@ -0,0 +1,11 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    user ||= User.new
+    if user.leader
```

Comment:
Acho que temos além de Lider, Admins e as autorizações deles são diferentes.

Reply:
No model de usuário está definido apenas o `collab` e o `lider`. Como podemos implementar o admin ?

Reply:
The user classes are: Leader (leader: true), collab (collab: true), normal_user(leader: false, collab: false). Leaders are supposed to have the ability to manage everything on the network. Collabs are supposed to be able to post something only. Normal Users are supposed just to consume the network content.

Reply:
Alright, I will make the changes.